api/generic-handlers.go

```diff
@@ -26,6 +27,12 @@ import (
 	"github.com/journeymidnight/yig/signature"
 )
 
+type MiddleWare interface {
```

Comment:
Who uses this MiddleWare?

Reply:
accessLogHandler, logHandler and the four handlers in generic-handlers